docs/Membership.md

```diff
@@ -23,45 +23,47 @@ Membership in working groups: T=technical working group, O=outreach working grou
- * James Malone (OBO Industry Liaison), SciBite, Cambridge, UK
+ * Rebecca Jackson (T), Intelligent Medical Objects (IMO), Rosemont, IL, USA
  * Nico Matentzoglu (T), Semanticly, Athens, Greece
- * [Chris Mungall](https://github.com/cmungall/), (T), Lawrence Berkeley National Laboratory, Berkeley, CA, USA
+ * [Chris Mungall](https://github.com/cmungall/) (T), Lawrence Berkeley National Laboratory, Berkeley, CA, USA
  * [Darren Natale](http://pir.georgetown.edu/pirwww/aboutpir/natalebio.shtml) (E), Georgetown University Medical Center, Washington DC, USA
- * David Osumi-Sutherland, EMBL-EBI, Cambridge, UK
+ * David Osumi-Sutherland (O), EMBL-EBI, Cambridge, UK
  * [James A. Overton](http://james.overton.ca) (T), [Knocean Inc.](http://knocean.com), Toronto, Canada
  * Bjoern Peters (E), La Jolla Institute for Immunology, La Jolla, CA, USA
- * [Philippe Rocca-Serra](https://eng.ox.ac.uk/people/philippe-rocca-serra/) (E, O), University of Oxford e-Research Centre, Department of Engineering Science, Oxford, UK.
- * [Alan Ruttenberg](http://sciencecommons.org/about/whoweare/ruttenberg/) (O, T), University at Buffalo, Buffalo, USA
+ * [Philippe Rocca-Serra](https://eng.ox.ac.uk/people/philippe-rocca-serra/) (O), University of Oxford e-Research Centre, Department of Engineering Science, Oxford, UK
+ * [Alan Ruttenberg](http://sciencecommons.org/about/whoweare/ruttenberg/) (O), University at Buffalo, Buffalo, USA
  * [Richard Scheuermann](https://www.jcvi.org/about/rscheuermann) (O), J. Craig Venter Institute, La Jolla, CA, USA
- * [Lynn Schriml](http://www.medschool.umaryland.edu/profiles/Schriml-Lynn/) (E, O), University of Maryland School of Medicine
+ * [Lynn Schriml](http://www.medschool.umaryland.edu/profiles/Schriml-Lynn/) (E, O), University of Maryland School of Medicine, Baltimore, MD, USA
  * Barry Smith (O), University at Buffalo, Buffalo, NY, USA
  * Chris Stoeckert (E, O), University of Pennsylvania, Philadelphia, PA, USA
- * [Nicole Vasilevsky](http://orcid.org/0000-0001-5208-3432) (E, T), Unversity of Colorado Anschutz Medical Campus, Auroa, CO, USA
+ * [Nicole Vasilevsky](http://orcid.org/0000-0001-5208-3432) (E, O), Unversity of Colorado Anschutz Medical Campus, Auroa, CO, USA
+ * Shawn Tan (T), European Bioinformatics Institute, Cambridge, UK 
  * Randi Vita (O), La Jolla Institute for Immunology, La Jolla, CA, USA
  * [Ramona Walls](http://www.cyverse.org/ramona-walls) (E, T), CyVerse, University of Arizona, Tucson, AZ, USA
- * [Jie Zheng](http://cbil.upenn.edu/profile-staff_bio/39) (T, O), University of Pennsylvania, Philadelphia, USA
+ * [Jie Zheng](http://cbil.upenn.edu/profile-staff_bio/39) (T, O), University of Pennsylvania, Philadelphia, PA, USA
  
 New members: follow the instructions on the [onboarding doc](https://docs.google.com/document/d/1MKhNTjZjGx6Ls72dybIV2ajYtbqtwP7O4lwxN2v3RBA/edit#heading=h.10q6n5qc13dp)
 
 Alumni:
 
  * Michael Ashburner
- * Colin Batchelor (E)
- * Eric Douglass (T)
```

Comment:
why remove this information about what committees people served on? should that be a secret?

Reply:
I feel it was unreliable and doesn't add much, especially for alumni. And only some of them even had that info.

Reply:
> Can you please add relevant start and end dates?

Not easily, no.

Reply:
at least for the people we're changing now we know the start and end dates. it would be nice to keep track of that going forwards

Reply:
Good idea to keep track. We know the start dates for the new additions. End dates are generally vague; I removed (alumnified) people who hadn't been active in some time, so it would be difficult to assign a specific end date. A removal date, sure.

Reply:
I don't think you need to remove alumni, ever. This is a valuable history for the community. Only transfer people from active to alumni need to get given an "end date"

Reply:
I didn't remove any alumni (at least not on purpose) -- I think those two showed up on the 'deleted' side because I deleted the (possibly spurious) committee labels (E) etc.

Reply:
Oh sorry I misread what you wrote. Glad we agree :)

Reply:
Cool. So can you approve this PR even though it's not 100% perfect? 🙂